README.md

```diff
@@ -33,10 +33,41 @@ All you have to do to get Accountant working is the following:
 (ns your-app-ns
   (:require [accountant.core :as accountant]))
 
-(accountant/configure-navigation!)
+(accountant/configure-navigation! {:nav-handler (fn [path] ...) :path-exists? (fn [path] ...)})
```

Comment:
Here's a question: is there a way we could make macros for `:nav-handler` and `:path-exists?` and then just allow a keyword option? i.e.:

```clojure
(accountant/configure-navigation! :secretary)
```

This sort of thing seems to work very well for secretary, but maybe less well with Bidi (since the example functions you provided include references to the global atom in addition to the function). 

Maybe we could do something instead of keywords where we chose to do either:
```clojure
{:router :secretary}
```
or 
```clojure
{:router :bidi
 :path-exists? (fn [path] ...)
 :nav-handler (fn [path] ...)}
```